releasenotes/notes/refactor-basicaer-to-basicprovider-e27aff9c8e81d26e.yaml

```
@@ -0,0 +1,91 @@
+---
+features:
+  - |
+    A new :mod:`qiskit.providers.basic_provider` module has been introduced
+    to replace ``qiskit.providers.basicaer``. This module contains provider
+    tools that mirror those of the ``BasicAer`` provider and offers a single,
+    non-efficient, statevector based simulator: :class:`.BasicSimulator`.
+    This simulator based on the :class:`.BackendV2` interface and is
+    exclusively intended for testing and simple prototyping, for more
+    advanced simulation capabilities, please refer to the ``qiskit-aer``
+    package.
+
+  - |
+    The :class:`.Target` interface and transpiler pipeline now support
+    target definitions with ``num_qubits=None``. This is to support the
+    creation of target-based simulators with a flexible number of qubits.
+    A target with ``num_qubits=None`` will exclusively contain global
+    instructions (with ``qargs=None``) and when given to the transpiler,
+    it is expected that the transpiler will not resize the circuit.
+    This change in the :class:`.Target` requires future transpiler passes
+    to account for the case where ``target.num_qubits is None``.
+
+upgrade:
+  - |
+    The ``qiskit.providers.basicaer`` module and all of its classes have been
+    removed following their deprecation on the 0.46 release. They have been
+    replaced by the new :mod:`qiskit.providers.basic_provider` and
+    the :mod:`qiskit.quantum_info` modules. The migration from using
```

This sounds like the `quantum_info` module is new, maybe rephrase like this?
```suggestion
    replaced by the :mod:`qiskit.quantum_info` module and the new 
    :mod:`qiskit.providers.basic_provider` module. The migration from using
```